ecl/hqlcpp/hqlstmt.cpp

```diff
@@ -824,6 +824,38 @@ HqlExprAssociation * BuildCtx::queryFirstCommonAssociation(AssocKind searchKind)
 }
 
 
+void BuildCtx::walkAssociations(AssocKind searchMask, IAssociationVisitor & visitor)
```

Comment:
Yes it is an unused function, but I retained it because it might be useful at a later date.